examples/python/src/booktest/models.py

```diff
@@ -0,0 +1,31 @@
+# Code generated by sqlc. DO NOT EDIT.
+from typing import List
+import datetime
+import enum
+
+import pydantic
+
+
+# Enums
+class BookType(str, enum.Enum):
```

Comment:
Really excited that we're using the built-in enum support.

Reply:
Me too! I was really happy the sql engines were able to properly handle enums, sometimes they can be tricky to use because `BookType.FICTION` is technically an instance of the `BookType` class and is sometimes serialized as `<BookType.FICTION: 'FICTION'>`.